tests/e2e/assertion-issuer-field.e2e.test.ts

```
@@ -0,0 +1,306 @@
+import { describe, it, expect, beforeAll, afterAll, beforeEach } from 'vitest';
+import request from 'supertest';
+import { setupTestApp } from './setup/setup-test-app';
+import type { Express } from 'express';
+import type { Shared, OB3 } from 'openbadges-types';
+
+describe('Assertion Issuer Field E2E Tests', () => {
+  let app: Express;
+  let testIssuer: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)